Cesium.Runtime/StdLibFunctions.cs

```diff
@@ -138,6 +138,17 @@ public static int Atoi(byte* ptr)
         return Convert.ToInt32(str);
     }
 
+    public static byte* GetEnv(byte* ptr)
+    {
+        var str = StdIoFunctions.Unmarshal(ptr);
+        if (str is null)
+        {
+            return null;
+        }
+
+        return StdIoFunctions.MarshalStr(Environment.GetEnvironmentVariable(str)?.TrimEnd('\\'));
```

Comment:
Why `TrimEnd('\\')`? I don't think it's normal for a runtime environment to strip trailing slashes from a variable like this.

Reply:
Similar to https://github.com/ForNeVeR/Cesium/pull/471#issuecomment-1807255956 I cannot understand why there different values in C / .NET, Maybe I should try to use other API? anyway I did not have any reasonable ideas, and put this "bad code" to show you the issues. Will try take a look.

Reply:
The API you use looks correct. What's the particular difference you observe? Or do you want me to take a look myself?

Reply:
I see one `\` at the end of PATH variable. Also on Linux we see `�t4��`. Maybe that's artifact of `printf`, don't know

Reply:
Regarding `\`, I believe it is normal: `PATH` variable (especially on Windows) in particular may just end with `\`, this is just its value.